zipline/assets/synthetic.py

```diff
@@ -82,11 +83,16 @@ def make_simple_equity_info(sids,
     num_assets = len(sids)
     if symbols is None:
         symbols = list(ascii_uppercase[:num_assets])
+
+    if names is None:
+        names = map(lambda x: str(x) + " INC.", symbols)
```

Comment:
My instinct is to use a list comprehension here, but definitely just a style choice.